OpenSim/Simulation/Model/Station.cpp

```diff
@@ -40,13 +40,21 @@ using SimTK::Vec3;
 /**
  * Default constructor.
  */
-Station::Station() :
-   ModelComponent()
+Station::Station() : Super()
```

Comment:
No need to change, but we don't use this convention in general (of using the `Super` typedef in constructors); we should be consistent.